content/basics/use-ably.textile

```diff
@@ -0,0 +1,123 @@
+ * You need to publish messages with very low latency
+ * You're running into your hourly API request limits
+
+The Ably Realtime interface is available in the most popular languages with "a number of SDKS":https://ably.com/download available.
+
+h3(#rest-interface). REST Interface
+
+The Ably REST interface provides a way for a wide range of server and client devices to communicate with the Ably service over REST. The REST API does not provide a realtime long-lived connection to Ably, but in all other respects is a simple subset of the full realtime messaging API.
+
+The primary use-case for the REST API is for servers that are part of the back-end of an application such as a web application, that publish messages, issue access tokens (temporary and client-specific) for its various clients, obtain message and presence history and retrieve statistics.
+
+CAVEAT-INPUT
+
+ Reasons to use the REST library:
+
+ * Your application server primarily issues tokens for clients and/or publish messages on channels
+ * Your application is mostly stateless
+ * You prefer synchronous requests, note that not all REST libraries are synchronous, but where the platform offers a synchronous and asynchronous approach, the REST libraries are more often synchronous
+
+The Ably REST interface is available in the most popular languages with "a number of SDKS":https://ably.com/download available.
+
+h2(#rest-api). Raw REST API
+
+Ably also provides a raw REST API where you don’t need to install any SDKs to use. 
+
+CAVEAT-INPUT
+
+ Reasons to use the raw REST API:
+
+ * Your chosen platform isn't supported
+ * You have resourcing constraints
+ 
+If you decide to use the REST API directly there is "extensive documentation":/api/rest-api/ to support you.
+
+h2(#protocol-adapters). Protocol Adapters
+
+The Ably platform also provides protocol adapters, providing open protocol support and interoperability with a huge array of third party client libraries supporting every imaginable platform.
+
+Although Ably SDKs are the recommended method for connecting to Ably, we also provide a number of protocol adapters as an alternative method for connecting to Ably. 
+
+The advantage to protocol adapters is that they require fewer resources and network overhead, however they do not support the full set of Ably features.
+
+We offer protocol adapters for:
+
+* "External MQTT libraries":#mqtt
+* "Ably SSE and EventSource API":#sse-eventsource
+* "Pusher and PubNub protocols":#pusher-pubnub
+
+h3(#mqtt). External MQTT libraries
+
+The Ably MQTT protocol adapter translates between MQ Telemetry Transport (MQTT) and Ably’s own protocol, allowing for seamless integration of any systems you may have. MQTT is a publish/subscribe, lightweight messaging protocol designed for constrained devices and low-bandwidth networks. One of the major uses of MQTT is with IoT (Internet of Things), where these principles are key to having effective communication between various devices.
```

Comment:
We could add a link to https://mqtt.org/

"where these principles are key" - what principles?